static/styles/one_course.css

```diff
@@ -0,0 +1,38 @@
+
+
+#one_course .card {
+    box-sizing: border-box;
+    width: 250px;
+    height: 254px;
+    background: rgba(238, 238, 238, 0.62);
+    border: 1px solid white;
+    box-shadow: 6px 6px 5px rgba(0, 0, 0, 0.22);
+    backdrop-filter: blur(6px);
+    border-radius: 17px;
+    text-align: left;
+    cursor: pointer;
+    transition: all 0.5s;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    user-select: none;
+    font-weight: bolder;
+    color: black;
+  }
+  
+#one_course  .card:hover {
+    border: 1px solid black;
+  }
+  
+#one_course .card:active {
+    transform: scale(0.95) rotateZ(1.7deg);
+  }
+
+#one_course a{
+    text-decoration: none;
+    color: inherit;
+}
+
+#one_course h5 {
+    color: #2E8B57;
+  }
```

Comment:
Missing one new line at the end of the file

Reply:
fixed